addons/base_automation/tests/test_automation_ui.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import HttpCase
+from odoo.tests.common import tagged
+
+@tagged('post_install', '-at_install')
+class TestAutomation(HttpCase):
```

Comment:
there exists a test file in the module test_base_automation, either in test_flow.py or test_tour.py

Reply:
bien vu